tools/mksurfdata_esmf/README

```diff
@@ -108,32 +130,3 @@ Running for the generation of multiple datasets
 NOTES
 ================
 
-------------------------------------
-slevis HAS LISTED ISSUES ENCOUNTERED
-------------------------------------
-0) New 30-sec raw data for soil texture fails. Try requesting more mem.
-1) Soil color & texture and ag fire peak month outputs too high in .log
-   TODO? Change frac_o from always 1.
-2) Pct lake has chged in the .log bc the old diagnostic omitted mask_i frac_o
-3) Slope, stddev have chged in the .log bc the old diag. used frac_o
-
---------------------------------------
-ALL raw dataset .nc FILES MUST BE cdf5
---------------------------------------
-> nccopy -k cdf5 oldfile newfile
--------------------------------------------------------------
-LAI raw dataset .nc FILE MUST HAVE "unlimited" time dimension
--------------------------------------------------------------
-ncks --mk_rec_dmn time file_with_time_equals_12.nc -o file_with_time_unlimited.nc
```

Comment:
I could see keeping the guidance for new raw datasets.

Reply:
OK, did that.